### PR TITLE
Migration fix

### PR DESCRIPTION
**Pull request by MDunitz**
## Need
- to be able to run migrations on a remote db

## Approach
- move migration code into dbctl script

### CODE REVIEW

File: dcpquery/db/__main__.py [hunk 1]
```diff
@@ -4,13 +4,14 @@
 
 import os, sys, argparse, logging, json
 
+import alembic
```
Comment (kislyuk) [inline dcpquery/db/__main__.py]: alembic doesn't seem to be used in this module, is that right? Consider removing unused import.

File: README.md [hunk 2]
```diff
@@ -228,9 +228,10 @@ Contributions are welcome; please read [CONTRIBUTING.md](CONTRIBUTING.md).
     - You can then edit the newly created migration file (in `dcpquery/alembic/versions`)
 
 ### Applying new migrations to the database
-- Ensure you are connected to the correct database (run `python -m dcpquery.db connect` to see the database url)
-- From the command line run `make apply-migrations`
-- To unapply a migration run `alembic downgrade migration_id` (the migration_id is the string in front of the underscore in the migration name, for file 000000000000_init_db.py the migration id is 000000000000)
+- Ensure you are connected to the correct database (run `python -m dcpquery.db connect` to see the database url, use the `--db remote` flag if necessary)
+- From the command line run `python -m dcpquery.db migrate` use the `--db remote` flag if necessary
+- To unapply a migration run (locally) `alembic downgrade migration_id` (the migration_id is the string in front of the underscore in the migration name, for file 000000000000_init_db.py the migration id is 000000000000)
```
Comment (kislyuk) [inline README.md]: Running alembic in this manner will also not work with the remote db, unfortunately. Perhaps we should add a generic wrapper for alembic, like `python -m dcpquery.db alembic COMMAND`, the same way as `python -m dcpquery.db run COMMAND` is a wrapper for `psql`. I can give you more pointers on how to do that if needed.

File: dcpquery/db/__main__.py (hunk 1 above)
Comment (MDunitz) [reply dcpquery/db/__main__.py]: 👍

File: README.md (hunk 2 above)
Comment (MDunitz) [reply README.md]: Discussed in person. Merging this pr and creating a ticket for that refactor

Comment (MDunitz) [reply README.md]: I think that makes sense. I'm going to merge this in and then create a ticket for that refactor

**Pull request merged**